config/routes.rb

```diff
@@ -599,7 +599,7 @@
 
   match 'host_wizard' => 'react#index', :via => :get
   constraints(id: /[^\/]+/) do
-    match 'experimental/hosts/:id' => 'react#index', :via => :get, :as => :host_details_page
+    match 'experimental/hosts/:id/*path' => 'react#index', :via => :get, :as => :host_details_page
```

Comment:
There has been a good point raised in the discourse thread:
What about routes that are ought to be served by backend like `/hosts/:id/clone` and so on?

Reply:
Adding a route that is already used by core would cause a clash regardless of whether the routing is handled on client or server. 

Similarly, having the following in a plugin is not a good idea:
```rb
# foreman_plugin/config/routes.rb
get '/hosts/:id/clone', to: 'plugin_hosts#clone'
```

Though the clash might not be immediately obvious in this case since each of the clashing routes is handled by routing in a different place (client vs server).